src/generate/renders/generated/CliCustom.ts

```diff
@@ -15,6 +15,7 @@ import { CodeGenConstants } from '../../../utils/models';
 import { CodeModelAz } from '../../CodeModelAz';
 import { HeaderGenerator } from '../Header';
 
+let allParams = [];
```

Comment:
why do we need a global parameter here?

Reply:
because body parameter construction logic is operation specific, the whole logic is an independent function https://github.com/Azure/autorest.az/pull/767/files#diff-424fdeac04a8fd4c32a0971332f877d31966db813c136c83ae17fa9b97b72bb7R516, we are trying to dedup at command layer, which is outside of this function's scope.